### PR TITLE
feat: Add support of Maps Static API

### PR DESCRIPTION
**Pull request by romavlasov**
issue: #343

### CODE REVIEW

**Comment by romavlasov** [comment]
Agree with you. Now it looks better.

File: googlemaps/maps.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        f.close()
+        ```
+    """
+
+    params = {"size": convert.size(size)}
+
+    if not markers:
+        if not (center or zoom is not None):
+            raise ValueError(
+                "both center and zoom are requered"
```
Comment (jpoehnelt) [inline googlemaps/maps.py]:
```suggestion
                "both center and zoom are required"
```

File: googlemaps/maps.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        if fillcolor:
+            self.params.append("fillcolor:%s" % fillcolor)
+
+        if geodesic:
+            self.params.append("geodesic:%s" % geodesic)
+
+        self.params.append(convert.location_list(points))
+
+
+def maps_download(client, size,
```
Comment (jpoehnelt) [inline googlemaps/maps.py]: what about `def static_map`? this makes it a little easier to link to all other documentation on the api

Comment (romavlasov) [reply googlemaps/maps.py]: I was thinking a lot about the name of this function and unfortunately the name "static_map" had not came to mind. I agree that it is more suitable name.

File: googlemaps/maps.py (hunk 1 above)
Comment (romavlasov) [reply googlemaps/maps.py]: updated

**Pull request merged**